webv2/web.go

```diff
@@ -653,6 +653,20 @@ func getTypeMap(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(filteredTypeMap)
 }
 
+// getErrors checks the errors in the spanner schema
+// and returns a list of tables with errors
+func getErrors(w http.ResponseWriter, r *http.Request) {
+	sessionState := session.GetSessionState()
+	var tableNames []string
+	for id, issues := range sessionState.Conv.SchemaIssues {
+		if len(issues.TableLevelIssues) != 0 {
+			tableNames = append(tableNames, sessionState.Conv.SpSchema[id].Name)
+		}
+	}
+	w.WriteHeader(http.StatusOK)
+	json.NewEncoder(w).Encode(tableNames)
+}
+
 // applyRule allows to add rules that changes the schema
```

Comment:
API name is `getErrors` but it returns a list of tables - This is confusing, consider renaming to `getTablesWithErrors()`.

Both the `Id` and the `Name` pair should be returned in this API for future extensibility.

Reply:
Makes sense. Done